config.json

```diff
@@ -4,15 +4,20 @@
 		"jqueryUi": "1.13.3"
 	},
 	"jqueryUi": [
+		{
+			"version": "1.14.0-beta.2",
+			"dependsOn": "jQuery 1.12+ / 2.2+ / 3.6+ / 4.0+",
+			"label": "Beta"
+		},
 		{
 			"version": "1.13.3",
-			"dependsOn": "jQuery1.8+",
+			"dependsOn": "jQuery 1.8+",
```

Comment:
There are now some spaces in the `dependsOn` value, is this on purpose?

Reply:
Yes. This string is not used in code but only displayed to a user; spaces make sense then. More specifically, they are displayed next to radio buttons for each UI version at https://jqueryui.com/download/. For example:
>  1.13.3 (Stable, for jQuery 1.8+)